src/main/java/org/kuali/kfs/kim/impl/role/RoleServiceImpl.java

```diff
@@ -233,7 +233,7 @@ public GenericQueryResults<DelegateMember> findDelegateMembers(final QueryByCrit
         return getCriteriaLookupService().lookup(DelegateMember.class, queryByCriteria, lc.build());
     }
 
-    @Cacheable(cacheNames = Role.CACHE_NAME, key = "'{" + Role.CACHE_NAME + "}id=' + #p0")
+    @Cacheable(cacheNames = Role.CACHE_NAME, key = "'namespaceCode=' + #p0 + '|' + 'name='+ #p1")
     @Override
     public RoleLite getRoleWithoutMembers(final String roleId) throws IllegalStateException {
```

Comment:
KualiCo may have introduced a bug with this method's annotation change, where the new cache key expects two namespace and name parameters even though the method only has a single ID parameter. We may want to change the key expression to `'id=' + #p0` for consistency with this class's get/put-role method changes, and I would suggest reporting this issue back to KualiCo.